srcpkgs/gnutls/template

```diff
@@ -19,6 +19,9 @@ homepage="https://gnutls.org"
 distfiles="https://www.gnupg.org/ftp/gcrypt/gnutls/v${version%.*}/gnutls-${version}.tar.xz"
 checksum=0ea8c3283de8d8335d7ae338ef27c53a916f15f382753b174c18b45ffd481558
 
+build_options="guile"
+build_options_default="guile"
```

Comment:
Nor here. Did you get it to cross compile with guile support? I had a lot of trouble making this work iirc.

Reply:
I will revert it back later.